### PR TITLE
3G-3G Relations

### PR DESCRIPTION
**Pull request by muralithiyagarajan**
3G-3G Relations
Motorola Nodes
Motorola Sites

### CODE REVIEW

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 1]
```diff
@@ -580,7 +587,62 @@ SELECT
...
 FROM ericsson_cm."RncFunction" t1
+union
+select
+    'MOTOROLA' AS "VENDOR",
+    t1.data->>'bss_name' AS "NODENAME"
+FROM motorola_cm."cell_x_export" t1
+
+`;
 
+const NETWORK_3G-3G_Relations = `
```
[the remark this answers is not in the document]
Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: Done

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 2]
```diff
@@ -627,7 +690,8 @@ VALUES
 		NOKIA_4G_KEY_PARAMAETERS : NOKIA_4G_KEY_PARAMAETERS,
 		NETWORK_CELLS : NETWORK_CELLS,
 		NETWORK_SITES : NETWORK_SITES,
-		NETWORK_NODES : NETWORK_NODES
+		NETWORK_NODES : NETWORK_NODES,
+		NETWORK_3G-3G_Relations : NETWORK_3G-3G_Relations
```
[the remark this answers is not in the document]
Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: Done

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 3]
```diff
@@ -524,6 +524,13 @@ UNION
 t1.data->>'userLabel' AS "SITENAME"
 from zte_cm."BtsSiteManager" t1
 UNION 
+-- ZTE 2G (xls)
+ SELECT 
+    'ZTE' as "VENDOR",
+    '2G' AS "TECH",
+t1.data->>'userLabel' AS "SITENAME"
+from zte_cm."SiteBaseBandShare" t1
```
[the remark this answers is not in the document]
Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: there is no MO in the name of BtsSiteManager in GSM_Radio Plan Template

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js (hunk 1 above)
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: const NETWORK_3G-3G_Relations -> `NETWORK_3G3G_RELATIONS`

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 4]
```diff
@@ -627,7 +690,8 @@ VALUES
 		NOKIA_4G_KEY_PARAMAETERS : NOKIA_4G_KEY_PARAMAETERS,
 		NETWORK_CELLS : NETWORK_CELLS,
 		NETWORK_SITES : NETWORK_SITES,
-		NETWORK_NODES : NETWORK_NODES
+		NETWORK_NODES : NETWORK_NODES,
+		NETWORK_3G3G_Relations : NETWORK_3G3G_Relations
```
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: Change **_Relations** to **_RELATIONS** for consistency with the other query names .i.e. keep it in uppercase.

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js (hunk 1 above)
Comment (erssebaggala) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: There was a hyphen you left behind in the variable name. `const NETWORK_3G-3G_Relations -> NETWORK_3G3G_RELATIONS`

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 5]
```diff
@@ -611,7 +676,8 @@ VALUES
 	('Nokia 4G parameters','Nokia 4G parameters', $$${NOKIA_4G_KEY_PARAMAETERS}$$, '{}', 'table',1, true),
 	('Network Cells','Network Cells', $$${NETWORK_CELLS}$$, '{}', 'table',2, true),
 	('Network Sites','Network Sites', $$${NETWORK_SITES}$$, '{}', 'table',2, true),
-	('Network Nodes','Network Nodes', $$${NETWORK_NODES}$$, '{}', 'table',2, true)
+	('Network Nodes','Network Nodes', $$${NETWORK_NODES}$$, '{}', 'table',2, true),
+	('Network 3G3G RELATIONS','Network 3G3G RELATIONS', $$${NETWORK_3G3G_Relations}$$, '{}', 'table',2, true)
```
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: You missed this **NETWORK_3G3G_Relations** to `NETWORK_3G3G_RELATIONS`

**Pull request merged**